examples/solana/build-staking-operation/main.go

```diff
@@ -5,59 +5,42 @@ import (
 	"fmt"
 	"log"
 	"math/big"
-	"os"
-	"path/filepath"
-	"strings"
 	"time"
 
 	"github.com/btcsuite/btcutil/base58"
+	"github.com/coinbase/coinbase-sdk-go/gen/client"
 	"github.com/coinbase/coinbase-sdk-go/pkg/coinbase"
-
 	bin "github.com/gagliardetto/binary"
 	"github.com/gagliardetto/solana-go"
 	"github.com/gagliardetto/solana-go/rpc"
 )
 
 var (
-	networkID          = "solana-devnet"
-	amount             = big.NewFloat(0.1)
-	rpcURL             = "https://api.devnet.solana.com"
-	defaultPrivKeyPath = filepath.Join(home(), ".config/solana/id.json")
+	networkID = client.NETWORKIDENTIFIER_SOLANA_MAINNET
+	amount    = big.NewFloat(0.1)
+
+	apiKeyFilePath   = "YOUR_API_KEY"
```

Comment:
Can we keep the method in which we source this information the same across examples?